tests/unit_tests/services/test_dds.py

```diff
@@ -386,27 +386,23 @@ def test_release_project_nomail(self):
             ])
 
     @gen_test
-    def test_get_dds_project_title(self):
-        mock_dds_project = [{
-                "Access": True,
-                "Last updated": "Fri, 01 Jul 2022 14:31:13 CEST",
-                "PI": "pi@email.com",
-                "Project ID": "snpseq00025",
-                "Size": 25856185058,
-                "Status": "In Progress",
-                "Title": "AB1234"
-                }]
+    def test_get_dds_version(self):
+        project_id = 'snpseq00001'
+        
+        dds_project = DDSProject(
+                dds_service=self.dds_service,
+                auth_token=self.token_file.name,
+                dds_project_id=project_id,
+                )
+        
 
-        with patch(
-                'delivery.models.project.DDSProject._run',
-                new_callable=AsyncMock,
-                return_value=json.dumps(mock_dds_project),
-                ):
-            dds_project = DDSProject(
-                    dds_service=self.dds_service,
-                    auth_token=self.token_file.name,
-                    dds_project_id=mock_dds_project[0]["Project ID"],
-                    )
+        yield dds_project.get_dds_version()
+        self.mock_dds_runner.run.assert_has_calls([
+                call([
+                        'dds',
+                        '--version', 
+                        ])
+                ])
+
```

Comment:
I would suggest that this unit test should also assert that the `get_dds_version` method correctly parses the version string it gets from dds. For that, you could for example patch the `._run` method similarly as in the deleted `test_get_dds_project_title` method and just have it return the expected output from the `dds --version` command